static/usr/lib/udev/rules.d/66-snapd-autoimport.rules

```diff
@@ -1,3 +1,2 @@
-# probe for assertions, must run before udisks2
-ACTION=="add", SUBSYSTEM=="block", KERNEL!="loop*", KERNEL!="ram*" \
-    RUN+="/usr/bin/unshare -m /usr/bin/snap auto-import --mount=/dev/%k"
+ACTION=="add|change", SUBSYSTEM=="block", KERNEL!="loop*", KERNEL!="ram*", ENV{SNAP_AUTO_IMPORT}!="0" \
+    ENV{SYSTEMD_WANTS}+="snapd.autoimport-device@%k.service"
```

Comment:
so to be clear `SYSTEMD_WANTS` here will trigger execution of a service named like `snapd.autoimport-device@foo.service` which then results in the ExecStart executing `snap auto-import --mount=/dev/foo` correct?

Reply:
Yes.

Reply:
But it is scheduled instead. udevd does not wait for it.